draft-ietf-taps-interface.md

```diff
@@ -2854,19 +2854,27 @@ large Message of indeterminate length.
 #### ReceiveError {#receive-error}
 
 ~~~
-Connection -> ReceiveError<messageContext, reason?>
+Connection -> ReceiveError<messageContext?, reason?>
```

Comment:
Why did you make the messageContext option? Don't you always need this to assign it to a message somehow?

Reply:
And you also removed the text below that said the messageContext would be optional (e.g. for partially received messages). Was that on purpose? However, I think for the two examples you mention below, it seems to make sense to provide some messageContext.

Reply:
I made it optional because, according to the text that was there, it was optional - it just wasn't written like that in the signature, which I thought was an oversight.

Then I removed the text that said it would be optional because I didn't see the added value of repeating what the function Event signature above already shows. I checked, the document doesn't consistently always say "and there is this parameter, and that parameter".

Regarding really making the messageContext optional or not... well... I can't see other uses than the two examples, and I agree that the messageContext makes sense to include in these two cases. But that doesn't mean that there couldn't be any other reception errors, where handing over a messageContext might not be possible. I don't know, personally, I can't think of any such case...

Reply:
```suggestion
Connection -> ReceiveError<messageContext, reason?>
```